cs130_back/handlers/listing.go

```diff
@@ -76,22 +76,6 @@ func CreateListing(db *gorm.DB, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if GetListingByID(db, &listing, w) == 0 {
```

Comment:
Can this be added back?

Reply:
I've added the getListingByID check. Do we need the other two? They were initially put because of the old way we implemented Users, Courses and Listings (everything owned everything)

Reply:
Yeah the GetListingByID check should be kept. Pretty sure we don't need the other two checks.